src/components/dataset/summary/row.vue

```diff
@@ -75,12 +75,16 @@ export default {
 @import '../../../assets/scss/_variables.scss';
 @import '../../../assets/scss/mixins';
 
-.dataset-row {
+.dataset-summary-row {
```

Comment:
Ah good call. Actually, I think this will fix issue 3 in the QA document: once loaded, these styles were getting applied to the rows of the dataset table.